src/biokbase/narrative/jobs/jobcomm.py

```diff
@@ -214,19 +214,25 @@ def _verify_job_id_list(self, req: JobRequest) -> None:
             )
             raise NoJobException("No valid job ids")
 
-    def start_job_status_loop(self, *args, **kwargs) -> None:
+    def start_job_status_loop(
+        self,
+        req: JobRequest = None,
+        init_jobs: bool = False,
+        cell_list: List[str] = None,
+    ) -> None:
         """
-        Starts the job status lookup loop. This runs every 10 seconds.
-        This has the bare *args and **kwargs to handle the case where this comes in as a job
-        channel request (gets a JobRequest arg), or has the "init_jobs" kwarg.
+        Starts the job status lookup loop. This runs every LOOKUP_TIMER_INTERVAL=10 seconds.
 
-        If init_jobs=True, this attempts to reinitialize the JobManager's list of known jobs
-        from the workspace.
+        :param req: first position is reserved for when this is invoked
+            as a job channel request (gets a JobRequest arg)
```

Comment:
Do you know where in the code this function gets called using the JobRequest object? I can't find anything in my local version of the codebase. I wonder if this is just cruft that can be removed.

Reply:
Oh. Yeah I cannot find a `start_update_loop` request type in the repo. Tricky.

Reply:
I'd go back in the git history and see if you can find changes to that function, and whether there was a point when the functionality changed but the documentation wasn't updated.

Reply:
Expert advice! One of these days I will make GitLens work for me.